source/docs/software/basic-programming/joystick.rst

```diff
@@ -69,27 +69,7 @@ The ``Joystick`` class is designed to make using a flight joystick to operate th
 
       exampleXbox = wpilib.XboxController(0) # 0 is the USB Port to be used as indicated on the Driver Station
 
-An example of how to use buttons on the ``XboxController``.
-
-.. tabs::
-
-   .. group-tab:: Java
-
-      .. rli:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/gearsbot/RobotContainer.java
-         :language: java
-         :lines: 39,85-88,96-99
-         :linenos:
-         :lineno-start: 39
-
-   .. group-tab:: C++
-
-      .. rli:: https://raw.githubusercontent.com/wpilibsuite/allwpilib/main/wpilibcExamples/src/main/cpp/examples/GearsBot/cpp/RobotContainer.cpp
-         :language: cpp
-         :lines: 41-48
-         :linenos:
-         :lineno-start: 41
-
-The ``XboxController`` class provides named indices for each of the buttons that you can access with ``XboxController.Button.kX.value``.  The rumble feature of the controller can be controlled by using ``XboxController.setRumble(GenericHID.RumbleType.kRightRumble, value)``.  Many users do a split stick arcade drive that uses the left stick for just forwards / backwards and the right stick for left / right turning.
+The ``XboxController`` class provides named methods for each of the buttons, and the indices can be accessed with ``XboxController.Button.kX.value``.  The rumble feature of the controller can be controlled by using ``XboxController.setRumble(GenericHID.RumbleType.kRightRumble, value)``.  Many users do a split stick arcade drive that uses the left stick for just forwards / backwards and the right stick for left / right turning.
```

Comment:
I think it would be helpful to have an example of the named method for each button (like ``getXButton``, ``getXButtonPressed`` and ``getXButtonReleased``. I was able to figure out what was meant, but other people may have a harder time.